src/lib.rs

```diff
@@ -1140,18 +1140,18 @@ where
 // TODO: https://github.com/Stebalien/term/issues/70
 enum AnyTerminal {
     /// Stdout terminal
-    Stdout(Box<term::StdoutTerminal>),
+    Stdout(Box<term::StdoutTerminal>, bool, bool, bool),
```

Comment:
Once you reach 2, it's better to name them with `Stdout { inner: Box<_>, reset: bool, ...`

Reply:
this is how you can tell i haven't written any rust in a while... lemme fix that